columnflow/tasks/reduction.py

```diff
@@ -467,54 +449,40 @@ def workflow_condition(self):
 
     @workflow_condition.create_branch_map
     def create_branch_map(self):
-        # DatasetTask implements a custom branch map, but we want to use the one in ForestMerge
-        return law.tasks.ForestMerge.create_branch_map(self)
+        # forward to super class (DatasetTask)
+        return super().create_branch_map()
 
-    def merge_workflow_requires(self):
-        return {
-            "stats": self.reqs.MergeReductionStats.req_different_branching(self),
-            "events": self.reqs.ReduceEvents.req_different_branching(self, branches=((0, -1),)),
-        }
+    def workflow_requires(self):
+        reqs = super().workflow_requires()
+        reqs["stats"] = self.reqs.MergeReductionStats.req_different_branching(self)
+        reqs["events"] = self.reqs.ReduceEvents.req_different_branching(self)
```

Comment:
Are we sure that we don't need the `branches` parameters here? I seem to remember that there was a minor issue with the resolution of specific branches - I think @mafrahm found that some time ago, maybe he can comment

Reply:
That's what the `req_different_branching` is actually doing. It prevents parameters from being passed that would imply an equal branch granularity.

Reply:
The issue @pkausw mentions was solved with this PR:  https://github.com/columnflow/columnflow/pull/544
and as far as I remember, setting this `branches` parameter was necessary to not submit certain jobs

Reply:
Indeed, this was required for the `merge_workflow_requires`, since this got called multiple times by workflows in different tree heights (in the "old" forest merge approach). This should not apply anymore with the forest merge removed :+1:

Thanks for the check!